orchestrator/migrations/versions/schema/2020-10-19_3323bcb934e7_fix_tsv_triggers.py

```diff
@@ -17,7 +18,8 @@
 def upgrade() -> None:
     conn = op.get_bind()
     conn.execute(
-        """
+        text(
+            """
```

Comment:
Side note, should we also update the migration generator to output migrations with the queries wrapped in `text()` ?

Reply:
Yes.  (Although I really really don't like the output that the migration generator is spitting out.)